examples/webgl_lines_fat.html

```diff
@@ -223,6 +226,54 @@
 
 			//
 
+			function onDocumentMouseMove( event ) {
+
+				if ( ! interactive ) return;
+
+				event.preventDefault();
+
+				var intersects = getIntersects( event.layerX, event.layerY );
+
+				if ( intersects.length > 0 ) {
+
+					if ( prevLinewidth === undefined ) {
+
+						prevLinewidth = matLine.linewidth
+
+					}
+
+					var res = intersects[ 0 ];
+
+					if ( res && res.object ) {
+
+						res.object.material.linewidth = prevLinewidth + prevLinewidth / 3;
+
+					}
+
+				} else if ( prevLinewidth !== undefined ) {
+
+					matLine.linewidth = prevLinewidth
+					prevLinewidth = undefined
+
+				}
+
+			}
+
+			var raycaster = new THREE.Raycaster();
+			var mouseVector = new THREE.Vector3();
+
+			function getIntersects( x, y ) {
+
+				x = ( x / window.innerWidth ) * 2 - 1;
+				y = - ( y / window.innerHeight ) * 2 + 1;
+
+				mouseVector.set( x, y, 0.5 );
+				raycaster.setFromCamera( mouseVector, camera );
+
+				return raycaster.intersectObject( scene, true );
```

Comment:
Perhaps you should pass in the `optionalTarget` in this case. The need for this arg was discussed in a previous PR. There may be different opinions on this.

Reply:
Could you link previous PR ?

Reply:
https://github.com/mrdoob/three.js/pull/12872

Reply:
Done in 3196781fb476